package.json

```diff
@@ -56,6 +56,7 @@
     "aws-sdk": "^2.1038.0",
     "babel-jest": "^29.0.0",
     "babel-plugin-inline-package-json": "^2.0.0",
+    "buffer": "^5.5.0",
```

Comment:
why is this needed?

Reply:
```suggestion
```

Reply:
It's added by yarn after e2e rebuilds for some reason. I can try to remove.